### ISSUE TITLE
Fix Request for UnconditionalOperationOnLogArgument: Spurious violations on chained operations

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#415]
```
@Slf4j
public final class DataUtil {

 public static void addTestData(URI uri, List<TransactionDocument> documents) {
        String data = documents.stream()
            .map(document -> document.toString() + "\n")
            .collect(Collectors.joining());
        log.info(WebClient.builder().build().post() // <--- 4x violation
            .uri(uri.resolve("/_bulk"))
            .header(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE)
            .body(BodyInserters.fromValue(data))
            .exchange()
            .flatMap(clientResponse -> clientResponse.bodyToMono(String.class))
            .block());
    }
}
```

**Comment by jborgers** [comment]
1 violation should cover them all
